docs/source/key_components/dashboard_visualization.md

```diff
@@ -0,0 +1,158 @@
+# Dashboard Visualization
+
+Env-dashboard is a post-experiment visualization tool, aims to provide useful guidance for researchers and provide users with intuitive data feature display.
```

Comment:
aims to provide more intuitive environment information, which will guide the design of the algorithm and continually fine-tuning.